package.json

```diff
@@ -74,7 +74,8 @@
     "wait-on": "^5.2.0",
     "webpack": "^5.10.0",
     "webpack-cli": "^4.2.0",
-    "webpack-node-externals": "^2.5.2"
+    "webpack-node-externals": "^2.5.2",
+    "whatwg-fetch": "^3.5.0"
```

Comment:
whats this dependency being used for?

Reply:
polyfill fetch for IE https://github.com/github/fetch

Reply:
👍